Remove mentions of Simple Content Access (#2745)
theforeman/foreman-documentation: guides/common/modules/con_managing-red-hat-subscriptions.adoc

> You can use future-dated subscriptions in a subscription allocation.

Reviewer: ```suggestion
You can use future-dated subscriptions in a subscription manifest.
```

On console.redhat.com they just call them "Manifests;" I think "subscription allocation" is the older term used on the deprecated access.redhat.com.

> When you add future-dated subscriptions to content hosts before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.

Reviewer: You no longer attach subscriptions to content hosts.

```suggestion
When you add future-dated subscriptions to your manifest before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.
```

> Attach the future-dated subscriptions to your content hosts before the current subscriptions expire.

Reviewer: You no longer attach subscriptions to content hosts.

```suggestion
```

> For more information, see xref:Attaching_Red_Hat_Subscriptions_to_Content_Hosts_{context}[].

Reviewer: This link (and the thing it links to) should be removed

> When a future-dated subscription becomes active, you must refresh the manifest to synchronize repository content.

Reviewer: I'm actually not 100% sure if this is true with SCA. @ceiu  ?

Reviewer: I guess if you add the subscriptions via the Katello interface, that automatically refreshes your manifest anyway. And there's no way to add subs to a manifest via console.redhat.com. So the only way you'd actually need a manifest refresh is if you added subscriptions via the deprecated access.redhat.com.

Reviewer 2: > I'm actually not 100% sure if this is true with SCA. @Ceiu ?

There may be some terminology confusion here, but if I'm following correctly: With SCA, it's no longer necessary for subscriptions to be attached to individual systems/hosts at all. So long as it's present in the org, and the desired content is promoted in the system's environment (repositories added to the host's content view), it should be visible.

As far as the subscription's active dates goes, even under SCA, Candlepin still considers the dates when determining what repos are active. Content provided by future-dated subscriptions will *not* be available to a host under SCA; but it will become available once the subscription becomes active.

Reviewer: > With SCA, it's no longer necessary for subscriptions to be attached to individual systems/hosts at all. So long as it's present in the org, and the desired content is promoted in the system's environment (repositories added to the host's content view), it should be visible.

Yes, this is my understanding as well :+1: 

> Candlepin still considers the dates when determining what repos are active. Content provided by future-dated subscriptions will _not_ be available to a host under SCA; but it will become available once the subscription becomes active.

And once the subscription becomes "active" (assuming that "active" means today's date is >= the subscription start date) it will become available without needing a manifest refresh, right?

Reviewer 2: > And once the subscription becomes "active" (assuming that "active" means today's date is >= the subscription start date) it will become available without needing a manifest refresh, right?

Correct. Depending on how smart the CP+subman logic is, the most the user or admin will need to do is a `subman refresh` to regenerate the SCA cert if it desyncs or otherwise doesn't pick up the now-active subscriptions automatically.

When you add future-dated subscriptions to content hosts before the expiry date of the existing subscriptions, you can have uninterrupted access to repositories.

Manually attach the future-dated subscriptions to your content hosts before the current subscriptions expire.

> Do not rely on the auto-attach method because this method is designed for a different purpose and might not work.

Reviewer: I remember seeing a Google doc covering this part; I think those modifications should be applied here.
